### PR TITLE
Update preview copy

### PR DESCRIPTION
**Pull request by codemonkey800**
https://preview-page-copy.vercel.app/

image

### CODE REVIEW

File: frontend/src/components/AppBar/metadataPreview.hooks.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export function useMetadataSections(): MetadataSection[] {
     {
       title: 'Name + descriptions',
```
Comment (neuromusic) [inline frontend/src/components/AppBar/metadataPreview.hooks.ts:55]:
```suggestion
      title: 'Describe what the plugin does',
```

File: frontend/src/components/AppBar/metadataPreview.hooks.ts [hunk 2]
```diff
@@ -68,7 +68,7 @@ export function useMetadataSections(): MetadataSection[] {
     {
       title: 'Support + contact',
```
Comment (neuromusic) [inline frontend/src/components/AppBar/metadataPreview.hooks.ts:69]:
```suggestion
      title: 'Tell users where they can get help',
```

File: frontend/src/components/AppBar/metadataPreview.hooks.ts [hunk 3]
```diff
@@ -84,13 +84,13 @@ export function useMetadataSections(): MetadataSection[] {
     {
       title: 'Development information',
```
Comment (neuromusic) [inline frontend/src/components/AppBar/metadataPreview.hooks.ts:85]:
```suggestion
      title: 'Give insight into your code',
```

File: frontend/src/components/AppBar/metadataPreview.hooks.ts [hunk 4]
```diff
@@ -84,13 +84,13 @@ export function useMetadataSections(): MetadataSection[] {
...
       description:
-        '[Explain what these metadata help the end user understand / why useful to populate them.]',
+        'Let users see your source code and know what kind of stability they should expect.',
       fields: getFields('version', 'license'),
     },
     {
       title: 'Implementation requirements',
```
Comment (neuromusic) [inline frontend/src/components/AppBar/metadataPreview.hooks.ts:91]:
```suggestion
      title: 'Specify system requirements',
```

**Review by neuromusic — CHANGES_REQUESTED** [review]
made suggestions for the headings for each section, too (these were pulled from the Copy doc)

https://docs.google.com/document/d/1EMfYUP2X2gstoYV-8wBrbfLJebpGHSf91TSxnWNeH8c/edit

can you also reorganize the fields to be under the headings as outlined in the Copy doc above?

**Comment by codemonkey800** [comment]
@neuromusic the fields have been updated!

image

**Review by neuromusic — APPROVED** [review]

**Pull request merged**